vue/components/ui/molecules/search-persistent/search-persistent.vue

```diff
@@ -272,13 +274,28 @@ export const SearchPersistent = {
             return `${filter.name}${filter.tenancy}${filter.context}`;
         },
         selectFilter(index) {
-            this.selectedFilter = index;
+            this.pendingSelectedFilter = index;
+
             this.valueData = this.filtersData[index].value;
         },
         isFilterSelected(index) {
             return index === this.selectedFilter;
         },
+        filterExists(filter) {
+            return this.filtersData.findIndex(
+                f => f.name === filter.name && f.tenancy === filter.tenancy
+            );
+        },
         addFilters(filters) {
+            for (let i = 0; i < filters.length; i++) {
+                const index = this.filterExists(filters[i]);
+
+                if (index >= 0) {
+                    this.updateFilter(filters[i], index);
+                    filters.splice(i, 1);
+                }
+            }

```

Comment:
What is this used for?

Reply:
To handle this: "Create filter with duplicated name = update existing filter"

This code checks for each newly created filter if it already exists. If yes updates it and removes him from the filters array so that they arent added again.